src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_VALUE = 25;
+    private Random random = new Random();
+    private ColorSupplier supplier = new ColorSupplier();
+
+    public Circle getRandomCircle() {
```

Comment:
All methods that are used only in the class must be private
```suggestion
    private Circle getRandomCircle() {
```